packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -1661,17 +1663,6 @@ class IrisGridTableModelTemplate<
       return false;
     }
 
-    // Editing the aggregations/totals which are floating above/below is not allowed
```

Comment:
As mentioned on our previous call, I think we still need this check - we need to make sure we don't paste and edit the aggregation row. If you have an input table and Add Aggregation, right now you can edit that aggregation row (though it's also not displaying the correct value when placed on the Bottom, which is another bug which should be fixed).

Reply:
Having this check will automatically fail cases where the pasting size is larger, given that the `endRow` will always be larger than the `topRowCount` + `table size` + `pendingRowCount` -- which is why I had initially opted to remove it, and then after we had spoken, I had tested further, but had not re-added it. With regards, to the aggregation row, is there any other way we can tell if a row is an aggregation row, and then based on that, I can have a check to basically check if the `endRow` is going beyond the aggregation row?

With regards to the separate bug you identified, as we discussed in call, I had created a seperate ticket for that: https://github.com/deephaven/web-client-ui/issues/2156

Reply:
Yea in terms of aggregation row we have a bit of a conflict there... it uses `floatingBottomRowCount` when aggregations are pinned to the bottom, and assumes the aggregations are the last rows in the table; but with these pending rows we actually want an "infinite" number of rows/always able to scroll. One of the downsides of the design of floating rows (as opposed to having it as a separate "table" of data for just the floating rows or something).

I think we'll just need to address that with #2156, but I won't hold it up for this (since the aggregations don't appear correctly already anyway).